public/usage-examples/graphics/fill_ellipse/fill_ellipse-2-extended.txt

```diff
@@ -0,0 +1,3 @@
+#### Clover Drawing 
+
+The following code shows exampels of using [Fill Ellipse](/api/graphics#fill-ellipse-3) to show how to draw a Clover drawing.
```

Comment:
I would suggest using:
"The following code shows an example of using [Fill Ellipse](/api/graphics#fill-ellipse-3) with different dimensions to draw a three-leaf clover."

(as the image is just a clover, rather than it showing _how_ to draw a clover)

Reply:
Thank you for fixing the issue regarding the usage example script and I have updated the description according to your requests!